app/spdb/build.sbt

```diff
@@ -573,6 +574,4 @@ def gsodb(version: Version) = AppConfig(
   bundles = List(
     BundleSpec(50, "edu.gemini.smartgcal.servlet", version)
   )
-) extending List(odbproduction(version), gsodbtest_credentials(version))
-
-
+) extending List(odbproduction(version), gsodb_credentials(version))
```

Comment:
I have no idea at all... I'll investigate a bit further since merging this PR is not an emergency priority.